nashvillefcc/nashvillefcc2020website
# Updated to tailwindcss v2 🎉

Resolves #102 

# Description of Changes

- [x] Replaced `@tailwindcss/custom-forms` with `@tailwindcss/forms`. See https://github.com/tailwindlabs/tailwindcss-custom-forms
- [x] Updated to `gatsby-plugin-postcss@^3`. See https://github.com/postcss/postcss/wiki/PostCSS-8-for-end-users#gatsby
- [x] Enabled tailwindcss jit. See https://tailwindcss.com/docs/just-in-time-mode

Colors can be imported directly from `tailwindcss/colors`.

I noticed we used `@tailwindcss/custom-forms` in /sponsors, but seems like the page is incomplete. There is a slight variant between the old `@tailwindcss/custom-forms` (top) and the new `@tailwindcss/forms` (bottom)

![image](https://user-images.githubusercontent.com/52666982/135920747-b3e1e041-a8cd-42bd-ba3e-8cf317003d01.png)

![image](https://user-images.githubusercontent.com/52666982/135920780-5171a516-7588-4f98-84de-ee025732f127.png)

## How Can I Test This?

Install the project and run. The project should compile successfully and running on the default port.
